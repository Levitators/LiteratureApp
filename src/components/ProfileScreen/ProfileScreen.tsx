
import React, { Component } from "react";
import { StyleSheet, Text, View } from "react-native";

export default class ProfileScreen extends Component<{}> {

  public render() {
    return (
      <View style={styles.container}>
        <Text>Profile Page</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    alignItems: "center",
    backgroundColor: "#F5FCFF",
    flex: 1,
    justifyContent: "center",
  },
});
