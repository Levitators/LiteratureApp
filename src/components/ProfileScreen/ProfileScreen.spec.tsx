import React from "react";
import renderer from "react-test-renderer";

import ProfileScreen from "./ProfileScreen";

it("Snapshot test", () => {
  const home = renderer.create(
    <ProfileScreen />,
  ).toJSON();
  expect(home).toMatchSnapshot();
});
