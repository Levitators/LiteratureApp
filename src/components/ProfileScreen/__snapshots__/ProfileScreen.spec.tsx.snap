// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Profile Page
  </Text>
</View>
`;
